WebDevStudios/wd_s
# Enables JIT mode

# WORK IN PROGRESS

### DESCRIPTION

This enables Tailwind JIT mode.

### OTHER


### STEPS TO VERIFY

1. Checkout
2. Run `npm i --legacy-peer-deps`
3. Run `npm run start` or `npm run watch`
4. Observe the blazing speeds

### DOCUMENTATION

Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?

I don't think so? Nothing about our workflow really changes here except things get faster.
